src/components/molecules/ScheduleEventListNG/utils.ts

```diff
@@ -0,0 +1,15 @@
+import { EVENT_STARTING_SOON_TIMEFRAME } from "settings";
+
+import { PersonalizedVenueEvent } from "types/venues";
+
+import { isEventLive, isEventStartingSoon } from "utils/event";
+
+export const isEventLater = (event: PersonalizedVenueEvent) =>
+  !(
+    isEventLive(event) ||
+    isEventStartingSoon(event, EVENT_STARTING_SOON_TIMEFRAME)
+  );
+
+export const isEventSoon = (event: PersonalizedVenueEvent) =>
+  !isEventLive(event) &&
+  isEventStartingSoon(event, EVENT_STARTING_SOON_TIMEFRAME);
```

Comment:
maybe would be better to move them to utils/events too? what do you think?

Reply:
fixed in 91ce25851e7b49d5cc1dc28c004ef21d37727e92